src/main/java/com/project/stockexchangeappbackend/rest/UsersController.java

```diff
@@ -319,8 +319,8 @@ public UserDTO getUser(@ApiParam(value = "Id of desired user.", required = true)
             "\n - yyyy-MM-ddThh:mm:ss.SSS-hh:mm \n - yyyy-MM-ddThh:mm:ss.SSS%2Bhh:mm (%2B means +)")
     @ApiResponses({@ApiResponse(code = 200, message = "Successfully paged and filtered user's transactions."),
             @ApiResponse(code = 403, message = "Access Denied."),
-            @ApiResponse(code = 400, message = "Id is not a number"),
-            @ApiResponse(code = 404, message = "User doesn't exist")})
+            @ApiResponse(code = 400, message = "Id is not a number", response = ErrorResponse.class),
```

Comment:
message = "The request could not be understood or was missing required parameters."